src/function_manager.jl

```diff
@@ -65,8 +65,10 @@ const FUNCTION_MANAGER = Ref{FunctionManager}()
 function _init_global_function_manager(gcs_address)
     @info "Connecting function manager to GCS at $gcs_address..."
     gcs_client = ray_jll.JuliaGcsClient(gcs_address)
-    ray_jll.Connect(gcs_client)
+    status = ray_jll.Connect(gcs_client)
+    ray_jll.ok(status) || error("Could not connect to GCS")
     FUNCTION_MANAGER[] = FunctionManager(; gcs_client, functions=Dict{String,Any}())
+    atexit(() -> ray_jll.Disconnect(gcs_client))
```

Comment:
Probably makes more sense to use a `finalizer` on `FunctionManager`

Reply:
added